apps/examples/board_specific_demo/ameba_secure_storage_demo/README.txt

```diff
@@ -7,6 +7,10 @@ examples/secure_storage
   This shows how to call write and read from Secure Storage with SE APIs,
   and how to read data written to flash.
 
+  This Example writes a user given byte of data (or default) to all bytes
+  of a user given slot (or default) of the Secure Storage
```

Comment:
Minor comment. Pls add full stop at end of line to maintain uniformity.

Reply:
Done